packages/mc-scripts/README.md

```diff
@@ -14,7 +14,30 @@ $ npm install --save @commercetools-frontend/mc-scripts
 
 ## Usage
 
-The package exports some functions to configure Webpack
+The CLI provides some commands. See https://docs.commercetools.com/custom-applications/development/available-scripts.
+
+## Using dotenv files (`.env`)
+
+Now the `mc-scripts` CLI has the dotenv features built-in.
+
+By default, the following dotenv files are loaded according to the current environment values specified on each command: `process.MC_APP_ENV` or `process.NODE_ENV`. The priority of how the files are merged and overwritten goes from top to bottom (highest defined variable overrides lower).
+
+- `.env.development.local`, `.env.test.local`, `.env.production.local`: Local overrides of environment-specific settings.
```

Comment:
The proxyCommand helper is called in build, start, compile-html and so forth if I read it right. I really like the implicit loading of env files which is part of the proxyCommand. 

What I am wondering is, if I build for production and have a .env.production.local defined it will be loaded and put onto the environment. Am I reading this right? Is that really intended if so?